skrf/vectorFitting.py

```diff
@@ -6,9 +6,9 @@
 if TYPE_CHECKING:
     from .network import Network
 
-from functools import wraps
 try:
     from . import plotting    # will perform the correct setup for matplotlib before it is called below
+    from .plotting import check_plotting
```

Comment:
Have you tested this line? This should not be possible, when no matplotlib is installed.

Reply:
Moved to `util.py`